src/components/GameBoard.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import Cell from "./Cell";
+
+const width = 7, height = 6;
```

Comment:
The variable name should be uppercase since `width` and `height` are the constants.